conan/tools/sbom/cyclonedx.py

```diff
@@ -1,16 +1,39 @@
 
-def cyclonedx_1_4(graph, **kwargs):
+
+def cyclonedx_1_4(conanfile, name=None, add_build=False, add_tests=False, **kwargs):
     """
-    (Experimental) Generate cyclone 1.4 sbom with json format
+    (Experimental) Generate cyclone 1.4 SBOM with JSON format
+
+    Creates a CycloneDX 1.4 Software Bill of Materials (SBOM) from a given dependency graph.
+
+
+
+    Parameters:
+        conanfile: The conanfile instance.
+        name (str, optional): Custom name for the metadata field.
+        add_build (bool, optional, default=False): Include build dependencies.
+        add_tests (bool, optional, default=False): Include test dependencies.
+
+    Returns:
+        The generated CycloneDX 1.4 document as a string.
+
+    Example usage:
+    ```
+    cyclonedx(conanfile.subgraph, name="custom_name", add_build=True, add_test=True)
```

Comment:
```suggestion
    cyclonedx(conanfile, name="custom_name", add_build=True, add_test=True, **kwargs)
```
The usage should always include the extra kwargs to be future safe